protocol/oracle-manager/contracts/modules/NodeModule.sol

```diff
@@ -214,7 +216,7 @@ contract NodeModule is INodeModule {
             nodeDefinition.nodeType == NodeDefinition.NodeType.STALENESS_CIRCUIT_BREAKER
         ) {
             //check if parents are processable
-            _processParentNodeOutputs(nodeDefinition);
+            _processParentNodeOutputs(nodeDefinition, new bytes32[](0), new bytes32[](0));
```

Comment:
this will fail if the parent node does a check on the validity of runtimeParams

Reply:
runtime args shouldn't be part of `isValid`, only the configured parameters.